fire/io/bernese/berneseset.py

```diff
@@ -0,0 +1,65 @@
+"""Dataklasser for et samlet sæt af beregninger (for en uge?), indsamlet fra ADDNEQ, CRD og COV output-filer fra
+Bernese. BerneseSet udgør et samlet datasæt for at oprette sager i FIRE. """
+
+import datetime
+from dataclasses import dataclass
+
+
+@dataclass
+class Kovarians:
+    """
+    Data indsamlet fra en datalinje i COV-output fil
+    """
+    station1: str
+    xyz1: str
+    station2: str
+    xyz2: str
+    flg: str
+    matrix_element: float
+
+
+@dataclass
+class Koordinatsaet:
+    """
+    Data indsamlet fra en datalinje i CRD-output fil
+    """
+    num: int
+    station_name: str
+    x: float
+    y: float
+    z: float
+    flag: str
```

Comment:
Prøv lige at tage et kik i word-dokumentet. Jeg har highlightet hvad jeg 99% sikker på der er det relevante information i cov-filen. For en specifik station bør det kun være seks tal der er relevante, altså den ene halvdel af den symmetriske 3x3 kovariansmatrix. 

Så fx for sektionen herunder er det kun relevant at kigge på de linjer hvor station1 == station2:
```
BOR1              X     BOR1              X         0.9326975128D-01
   
BOR1              Y     BOR1              X         0.2008472992D-01
BOR1              Y     BOR1              Y         0.1634563313D-01
   
BOR1              Z     BOR1              X         0.9670543502D-01
BOR1              Z     BOR1              Y         0.2236056402D-01
BOR1              Z     BOR1              Z         0.1607287392D+00

BUDP              X     BOR1              X        -0.1649457919D-02
BUDP              X     BOR1              Y        -0.1629805447D-02
BUDP              X     BOR1              Z        -0.2554317586D-03
BUDP              X     BUDP              X         0.9145031116D-01
   
BUDP              Y     BOR1              X        -0.9737721161D-03
BUDP              Y     BOR1              Y        -0.3136529871D-03
BUDP              Y     BOR1              Z        -0.4867097071D-03
BUDP              Y     BUDP              X         0.1754111808D-01
BUDP              Y     BUDP              Y         0.1860172410D-01
   
BUDP              Z     BOR1              X        -0.1546795149D-02
BUDP              Z     BOR1              Y        -0.1652294653D-02
BUDP              Z     BOR1              Z         0.2451237273D-02
BUDP              Z     BUDP              X         0.9706757931D-01
BUDP              Z     BUDP              Y         0.2230707662D-01
BUDP              Z     BUDP              Z         0.1740501496D+00
```

Gør vi det har vi så sæt af 6 tal for hver station: `XX`, `YX`, `YY`, `ZX`, `ZY` og `ZZ`. Det er DEM det er relevant at gemme på - de resterende kan vi se bort fra.

Reply:
Det betyder så også at jo længere ned i filen du kommer, jo mere skal der sorteres fra :-) Men det skulle gerne kunne resultere i en `Kovarians` med 6 tal (3 varianser, 3 kovarianser) pr. station i beregningen.

Reply:
Got it. Resultatet af parsing kan dermed blive væsentligt mindre end hvis alle elementer skulle med.

Reply:
Jep. Og skulle jeg tage fejl, og det hele er relevant, så er der lidt mere at holde styr på 😬